rulesets/.eslintignore

```diff
@@ -0,0 +1,2 @@
+*.js
```

Comment:
We should remove this so it lints the JS files

Reply:
@travisgosselin, I usually see the eslint config at the same level as the package.json. Regardless, it makes no difference where its located, the linting works the same. If there was a .eslintrc file higher up in the folder structure, this one would still take precedence.

Reply:
Yep same level as package.json makes sense.